OSX El Capitan 10.11.6 compilation issue

KeepassXC from Git 
2.1.4-325-g943c315

OSX Version 10.11.6
clang: Apple LLVM version 8.0.0 (clang-800.0.42.1)
Target: x86_64-apple-darwin15.6.0

I followed this to set up the build environment: 
https://github.com/keepassxreboot/keepassxc/wiki/Set-up-Build-Environment-on-OS-X

and this for build instructions
https://github.com/keepassxreboot/keepassxc/wiki/Building-KeePassXC

I did brew update && brew upgrade

The compilation fails with this:

```
Building CXX object src/autotype/mac/CMakeFiles/keepassx-autotype-cocoa.dir/AutoTypeMac.cpp.o
/Users/till_old/build/keepassxc/src/autotype/mac/AutoTypeMac.cpp:406:18: error: cannot initialize a variable of type
      'CGEventFlags' with an rvalue of type 'int'
    CGEventFlags nativeModifiers = 0;
                 ^                 ~
/Users/till_old/build/keepassxc/src/autotype/mac/AutoTypeMac.cpp:408:18: error: cannot initialize a variable of type
      'CGEventFlags' with an rvalue of type '(anonymous enum at
      /System/Library/Frameworks/Carbon.framework/Frameworks/HIToolbox.framework/Headers/Events.h:120:1)'
    CGEventFlags shiftMod = shiftKey;
                 ^          ~~~~~~~~
/Users/till_old/build/keepassxc/src/autotype/mac/AutoTypeMac.cpp:409:18: error: cannot initialize a variable of type
      'CGEventFlags' with an rvalue of type '(anonymous enum at
      /System/Library/Frameworks/Carbon.framework/Frameworks/HIToolbox.framework/Headers/Events.h:120:1)'
    CGEventFlags cmdMod = cmdKey;
                 ^        ~~~~~~
/Users/till_old/build/keepassxc/src/autotype/mac/AutoTypeMac.cpp:410:18: error: cannot initialize a variable of type
      'CGEventFlags' with an rvalue of type '(anonymous enum at
      /System/Library/Frameworks/Carbon.framework/Frameworks/HIToolbox.framework/Headers/Events.h:120:1)'
    CGEventFlags optionMod = optionKey;
                 ^           ~~~~~~~~~
/Users/till_old/build/keepassxc/src/autotype/mac/AutoTypeMac.cpp:411:18: error: cannot initialize a variable of type
      'CGEventFlags' with an rvalue of type '(anonymous enum at
      /System/Library/Frameworks/Carbon.framework/Frameworks/HIToolbox.framework/Headers/Events.h:120:1)'
    CGEventFlags controlMod = controlKey;
                 ^            ~~~~~~~~~~
5 errors generated.
make[2]: *** [src/autotype/mac/CMakeFiles/keepassx-autotype-cocoa.dir/AutoTypeMac.cpp.o] Error 1
make[1]: *** [src/autotype/mac/CMakeFiles/keepassx-autotype-cocoa.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/autotype/mac/AutoTypeMac.cpp
+++ b/src/autotype/mac/AutoTypeMac.cpp
@@ -403,12 +403,12 @@ uint16 AutoTypePlatformMac::qtToNativeKeyCode(Qt::Key key)
 //
 CGEventFlags AutoTypePlatformMac::qtToNativeModifiers(Qt::KeyboardModifiers modifiers, bool native)
 {
-    CGEventFlags nativeModifiers = 0;
+    CGEventFlags nativeModifiers = CGEventFlags(0);
 
-    CGEventFlags shiftMod = shiftKey;
-    CGEventFlags cmdMod = cmdKey;
-    CGEventFlags optionMod = optionKey;
-    CGEventFlags controlMod = controlKey;
+    CGEventFlags shiftMod = CGEventFlags(shiftKey);
+    CGEventFlags cmdMod = CGEventFlags(cmdKey);
+    CGEventFlags optionMod = CGEventFlags(optionKey);
+    CGEventFlags controlMod = CGEventFlags(controlKey);
 
     if (native) {
         shiftMod = kCGEventFlagMaskShift;

